<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/window/settings</normaloff>:/window/settings</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="twMain">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabApplication">
      <attribute name="title">
       <string>Application</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="hl1">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="vl1">
           <item>
            <widget class="QLabel" name="labelLanguage">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Language</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStyle">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Style</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelToolbarStyle">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Toolbar style</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vl2">
           <item>
            <widget class="QComboBox" name="cbLanguage">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbStyle">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbToolbarStyle">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="hs1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl2">
         <item>
          <widget class="QLabel" name="labelExternalPlayer">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>External player</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eExternalPlayer">
           <property name="text">
            <string notr="true">xdg-open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bExternalPlayerDefault">
           <property name="text">
            <string notr="true">Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDatabase">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="hl3">
         <item>
          <widget class="QLabel" name="labelDatabaseFile">
           <property name="text">
            <string>Database file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eDatabaseFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="bOpenDatabaseFile">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cCheckFilesAtStartup">
         <property name="text">
          <string>Checking availability of files at startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vs2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>118</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFilmsList">
      <attribute name="title">
       <string>Films list</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="hl4">
         <item>
          <layout class="QVBoxLayout" name="vl3">
           <item>
            <widget class="QCheckBox" name="cHighlightViewed">
             <property name="text">
              <string>Highlight the viewed films</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cHighlightFavourite">
             <property name="text">
              <string>Highlight the favourite films</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cHighlightUnavailable">
             <property name="text">
              <string>Highlight the unavailable films</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vl4">
           <item>
            <widget class="QPushButton" name="bSelectColorViewed">
             <property name="text">
              <string>Select color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bSelectColorFavourite">
             <property name="text">
              <string>Select color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bSelectColorUnavailable">
             <property name="text">
              <string>Select color</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="hs2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>75</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
